src/System.Windows.Forms.Primitives/src/Windows/Win32/PInvoke.GetThreadDpiAwarenessContext.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Windows.Forms;
+
+namespace Windows.Win32
+{
+    internal static partial class PInvoke
+    {
+        /// <summary>
+        ///  Tries to get the thread dpi awareness context.
+        /// </summary>
+        /// <returns>
+        /// Returns the thread dpi awareness context if the API is available in this version of OS.
+        /// Otherwise, returns <see cref="DPI_AWARENESS_CONTEXT.UNSPECIFIED_DPI_AWARENESS_CONTEXT"/>.
+        /// </returns>
```

Comment:
It doesn't _try_, it _gets_ it :) If it tried, the API would be something like this: `bool TryGetThreadDpiAwarenessContextInternal(out DPI_AWARENESS_CONTEXT)`.

```suggestion
        /// <summary>
        ///   Gets the thread dpi awareness context.
        /// </summary>
        /// <returns>
        ///  The thread dpi awareness context if the API is available in this version of OS.
        ///  Otherwise, <see cref="DPI_AWARENESS_CONTEXT.UNSPECIFIED_DPI_AWARENESS_CONTEXT"/>.
        /// </returns>
```